sirepo/template/template_common.py

```diff
@@ -230,7 +230,7 @@ def validate_model(model_data, model_schema, enum_info):
                 'BeamList', 'MirrorFile', 'ImageFile', 'String', 'OptionalString', 'MagneticZipFile',
```

Comment:
This list should be alphabetical and line by line so it can be easily sorted. Otherwise, too easy to get duplicates and/or miss something. Probably would define as a constant.